docs/manual/variables.ipynb

```diff
@@ -87,7 +87,7 @@
    "metadata": {},
    "source": [
     "The `name` variable is mutable, so you can change it later, but if you try to\n",
-    "change `id` after it's initialized, you'll get a compiler error. (You can\n",
+    "change `user_id` after it's initialized, you'll get a compiler error. (You can\n",
```

Comment:
note the [variable is named user_id on L81](https://github.com/modularml/mojo/pull/1447/commits/a8120c2f22d11cf9844db7f60f407ee44ddb7711#diff-f6dcceb293d5fecc97bec57fcdc3485e21c3a97e9221939af5fdf4e118dcd899L81)